setup.py

```diff
@@ -1,9 +1,15 @@
 from setuptools import setup, find_packages
+import sys
+
+if sys.version_info < (3, 2) and "install" in str(sys.argv):
+    sys.exit('rpc-component requires Python >= 3.2 '
```

Comment:
rpc_component

Reply:
Originally I was going to use rpc_component, but then I decided to write the exit message ("rpc-component...") to match the output printed via `pip install .`, which is `rpc-component requires Python '>=3.2' but the running Python is 2.7.15`.  This also matches the "component name" option specified in the README.md file.  Also, having a minimum version reference crossed my mind but I held off in favor of having the script appear shorter and more simple.

Reply:
I guess pip has a thing about underscores.

The text in the readme is unrelated to this, those references are to different projects.